py/desispec/validredshifts.py

```diff
@@ -0,0 +1,152 @@
+# Example usage:
+# redrock_path = '/global/cfs/cdirs/desi/spectro/redux/guadalupe/tiles/cumulative/1392/20210517/redrock-5-1392-thru20210517.fits'
+# cat = validate(redrock_path, return_target_columns=True)
+
+import os, warnings
+import numpy as np
+from astropy.table import Table, hstack, join
+import fitsio
+
+
+def get_good_fiberstatus(cat, isqso=False):
+    if not isqso:
+        good_fiberstatus = cat['COADD_FIBERSTATUS']==0
+    else:
+        good_fiberstatus = (cat['COADD_FIBERSTATUS']==0) | (cat['COADD_FIBERSTATUS']==8388608) | (cat['COADD_FIBERSTATUS']==16777216)
```

Comment:
here and elsewhere, please use `desispec.maskbits.fibermask` etc. to create the masks by name instead of hardcoding magic numbers like 8388608 .